README.md

```diff
@@ -1,2 +1,46 @@
-# math-magicians
-A react app that has calculator feature
+`React` `ES6` `NPM` `LINTERS` `GIT FLOW` <br>
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo List Project
```

Comment:
- [ ] Please it will nice to update your readme file to reflect the changes for this milestone, as this will make your readme file more informative and professional, kindly update your project title. Also remember to add license information, this [template](https://github.com/microverseinc/readme-template) will be a good guide.